CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # nextflow-io/nf-schema: Changelog
 
+# Unreleased
+
+## Bug fixes
+
+1. Resolved an issue where the UniqueEntriesEvaluator did not correctly detect non-unique combinations.
```

Comment:
Could you add this to the `v2.1.1` changelog? That's the next version :p

Reply:
Whoops, sorry..! Yes you're right. Should be good now hopefully.